### PR TITLE
feature/519_python_clients_for_hive

### PR DESCRIPTION
**Pull request by frbattid**
* Implement issue #519 
* No unit tests required
* (unofficial) e2e tests passed:
```
$ python hiveserver1-client.py cosmos.lab.fiware.org 10000 default frb XXXXXXXX
remotehive>
$ python hiveserver2-client.py cosmos.lab.fiware.org 10000 default frb XXXXXXXX
remotehive>
```
* Assignee @fgalan

### CODE REVIEW

File: resources/README.md [hunk 1]
```diff
@@ -9,11 +9,15 @@
...
 ##<a name="section1"></a>Hive clients
 
+# NGSI Notifications
 A set of Hive clients mimicing the native Hive CLI written in several programming languages both for HiveServer1 and HiveServer2. Specifically, there are clients for:
 
 * Java:
     * [`hiveserver1-client`](./hiveclients/java/hiveserver1-client)
     * [`hiveserver2-client`](/.hiveclients/java/hiveserver2-client)
+* Python:
+    * [`hiveserver1-client.py`](./hiveclients/java/hiveserver1-client.py)
```
Comment (fgalan) [inline resources/README.md]: Is the path `./hiveclients/java/hiveserver1-client.py` corrrect? Maybe 'java' should be 'python'...

File: resources/hiveclients/python/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#Python-based clients for HiveServer1 and HiveServer2
+
+##HiveServer1 client
+This sample code works with [`python_hive_utils`](https://github.com/eventbrite/python_hive_utils). The sample code given by the [Hive official](https://cwiki.apache.org/confluence/display/Hive/HiveClient#HiveClient-Python page) only works on a standalone server where is possible to access the built Python libraries for Hive.
+
+Thus, start by installing `python_hive_utils`:
+
+    $ pip install hive_utils
```
Comment (fgalan) [inline resources/hiveclients/python/README.md]: Maybe is better to state that root permissions are needed with:

```
sudo pip install hive_utils
```

File: resources/hiveclients/python/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    
+In order to run the client, just execute the `hiveserver1-client` script:
+
+    $ python hiveserver1-client <hive_host> <hive_port> <db_name> <hadoop_user> <hadoop_password>
+
+##HiveServer2 client
+This sample code works with [`pyhs2`](https://github.com/BradRuderman/pyhs2), thus start by installing it:
+
+    $ pip install pyhs2
```
Comment (fgalan) [inline resources/hiveclients/python/README.md]: sudo pip...

File: resources/hiveclients/python/hiveserver1-client.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# imports
+import sys
+import hive_utils
+from hive_service.ttypes import HiveServerException
+
+# get the input parameters
+hiveHost = sys.argv[1]
```
Comment (fgalan) [inline resources/hiveclients/python/hiveserver1-client.py]: Maybe you could check that the argv array has the right number of argument (no more, no less) and raise and error (and print the usage hint) in that case.

File: resources/hiveclients/python/hiveserver2-client.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# imports
+import sys
+import pyhs2
+from pyhs2.error import Pyhs2Exception
+
+# get the input parameters
```
Comment (fgalan) [inline resources/hiveclients/python/hiveserver2-client.py]: Maybe you could check that the argv array has the right number of argument (no more, no less) and raise and error (and print the usage hint) in that case.

File: resources/hiveclients/python/hiveserver1-client.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```
Comment (fgalan) [inline resources/hiveclients/python/hiveserver1-client.py]: License header missing?

File: resources/hiveclients/python/hiveserver2-client.py [hunk 7]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```
Comment (fgalan) [inline resources/hiveclients/python/hiveserver2-client.py]: License header missing?

Comment (frbattid) [inline resources/hiveclients/python/hiveserver2-client.py]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c

File: resources/hiveclients/python/hiveserver1-client.py (hunk 6 above)
Comment (frbattid) [inline resources/hiveclients/python/hiveserver1-client.py]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c

File: resources/hiveclients/python/hiveserver2-client.py (hunk 5 above)
Comment (frbattid) [inline resources/hiveclients/python/hiveserver2-client.py]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c

File: resources/hiveclients/python/hiveserver1-client.py (hunk 4 above)
Comment (frbattid) [inline resources/hiveclients/python/hiveserver1-client.py]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c

File: resources/README.md (hunk 1 above)
Comment (frbattid) [inline resources/README.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c

File: resources/hiveclients/python/README.md (hunk 2 above)
Comment (frbattid) [inline resources/hiveclients/python/README.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c

File: resources/hiveclients/python/README.md (hunk 3 above)
Comment (frbattid) [inline resources/hiveclients/python/README.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/3da5ee39ebc1b9baf11128573c3db2e1d277669c

File: resources/hiveclients/python/hiveserver1-client.py [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+# For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+
+# imports
+import sys
+import hive_utils
+from hive_service.ttypes import HiveServerException
+
+# get the input parameters
+if len(sys.argv) != 6:
+    print 'Usage: python hiveserve1-client.py <hive_host> <hive_port> <db_name> <hadoop_user> <hadoop_password>'
```
Comment (fgalan) [inline resources/hiveclients/python/hiveserver1-client.py]: A 'r' is missing

File: resources/hiveclients/python/hiveserver2-client.py [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+# For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+
+# imports
+import sys
+import pyhs2
+from pyhs2.error import Pyhs2Exception
+
+# get the input parameters
+if len(sys.argv) != 6:
+    print 'Usage: python hiveserve2-client.py <hive_host> <hive_port> <db_name> <hadoop_user> <hadoop_password>'
```
Comment (fgalan) [inline resources/hiveclients/python/hiveserver2-client.py]: A 'r' is missing

File: resources/hiveclients/python/hiveserver1-client.py (hunk 8 above)
Comment (frbattid) [inline resources/hiveclients/python/hiveserver1-client.py]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/cfc371eee3d79575e3021bb4c7772636112380ae

File: resources/hiveclients/python/hiveserver2-client.py (hunk 9 above)
Comment (frbattid) [inline resources/hiveclients/python/hiveserver2-client.py]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/cfc371eee3d79575e3021bb4c7772636112380ae

**Comment by fgalan** [comment]
LGTM

**Pull request merged**